docs/server/production-deploy.md

```diff
@@ -68,20 +68,38 @@ Every shard is low contention by design and it is very difficult to oversubscrib
 With that said, here are some guidelines to some common bottlenecks:
 
 - **Database**. The vast majority of the time the database will be the bottleneck. **We highly recommend setting alerts on `ScheduleToStart` latency** to look out for this. Also check if your database connection getting saturated.
-- **Internal services**. The next layer will be scaling the 4 internal services of Temporal ([Frontend, Matching, History, and Worker](https://docs.temporal.io/docs/server-architecture/)). Monitor each accordingly. The Frontend service is more CPU bound, whereas the History and Matching services require more memory.
+- **Internal services**. The next layer will be scaling the 4 internal services of Temporal ([Frontend, Matching, History, and Worker](https://docs.temporal.io/docs/server-architecture/)).
+  Monitor each accordingly. The Frontend service is more CPU bound, whereas the History and Matching services require more memory.
+  If you need more instances of each service, spin them up separately with different command line arguments. You can learn more cross referencing [our Helm chart](https://github.com/temporalio/helm-charts) with our [Server Configuration reference](https://docs.temporal.io/docs/server/configuration/).
 - See the **Server Limits** section below for other limits you will want to keep in mind when doing system design, including event history length.
 - [Multi-Cluster Replication](https://docs.temporal.io/docs/server/multi-cluster/) is an experimental feature you can explore for heavy reads.
 
+Finally you want to set up alerting and monitoring on Worker metrics.
+When Workers are able to keep up, `ScheduleToStart` latency is close to zero.
+The default is 4 Workers (aka pollers, as the Workers poll Task Queues), which should handle no more than 300 messages per second.
+
+Specifically, the primary scaling metrics are located in the server's dynamic configs:
+
+- `MaxConcurrentActivityExecutionSize`: Defaults to 1000
+- `MaxConcurrentWorkflowTaskExecutionSize`: Defaults to 1000
+- `MaxConcurrentActivityTaskPollers`: Defaults to 4
+- `MaxConcurrentWorkflowTaskPollers`: Defaults to 4
```

Comment:
TODO: this is actually overridden by SDK. server and SDK defaults differ for some reason.

Reply:
https://github.com/temporalio/temporal/blob/fe05751305b1cb50b68efa23f8aa5f1b34f45bc5/service/worker/service.go#L121

https://github.com/temporalio/sdk-java/blob/bef967639fcdbe14ca37b80ac816596412846e5f/temporal-sdk/src/main/java/io/temporal/worker/WorkerOptions.java#L51

```suggestion
Specifically, the primary scaling metrics are located in the server's dynamic configs:

- `MaxConcurrentActivityTaskPollers` and `MaxConcurrentWorkflowTaskPollers`: [Defaults to 4](https://github.com/temporalio/temporal/blob/fe05751305b1cb50b68efa23f8aa5f1b34f45bc5/service/worker/service.go#L121)
- `MaxConcurrentActivityExecutionSize` and ``MaxConcurrentWorkflowTaskExecutionSize`: [Defaults to 200](https://github.com/temporalio/sdk-java/blob/bef967639fcdbe14ca37b80ac816596412846e5f/temporal-sdk/src/main/java/io/temporal/worker/WorkerOptions.java#L51)
```